chainer/function_node.py

```diff
@@ -589,3 +591,225 @@ def delete_hook(self, name):
 
         """
         del self.local_function_hooks[name]
+
+
+def grad(outputs, inputs, grad_outputs=None, grad_inputs=None, set_grad=False,
```

Comment:
Please add this to the document. According to #3117, I think add this function to `docs/source/reference/core/function.rst` is appropriate?

Reply:
Oh, I forgot it. Will do. As you wrote, `function.rst` looks a good place.